### DISCUSSION TITLE
分布式一致性与共识算法 · /consensus

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#518] [Announcements]
https://draveness.me/consensus

**Comment by ericchuhong** [comment]
请问下绘图用的工具是什么工具

**Comment by draveness** [comment]
> @ericchuhong
> 请问下绘图用的工具是什么工具

看看最后

**Comment by ericchuhong** [comment]
> @Draveness
> > @ericchuhong
> > 请问下绘图用的工具是什么工具
> 
> 看看最后

嗯嗯 sketch 谢谢分享

**Comment by lanhu96** [comment]
对区块链一直有关注，方便加个微信吗huangliangxin

**Comment by foreseaz** [comment]
勘误：
「在 1998 年的秋天，加州伯克利大学的教授 Eric Brewer 第一次发布了 CAP 理论，在 1999 年论文... 」
CAP 理论是 Eric Brewer 教授在 PODC 2000 研讨会提出，2002 年被 Seth Gilbert 和 Nancy Lynch 证明为定理，也就是引用的这篇论文 :D

**Comment by orangle** [comment]
竟然没有 2pc 。。

**Comment by cicilaw** [comment]
您好，看到您的文章质量非常高，想邀请您成为虫洞社区签约作者。虫洞社区是中美区块链技术学习社区。虫洞社区鼓励内容生产者产生高质量内容，并给予合理的回报，也希望能帮助内容消费者获得高质量的区块链内容，并让数字货币投资者获得有价值的投资洞见。同时，虫洞社区已经积累了大量的区块链深度从业者，便于作者建立个人品牌。不知道是否方便加您微信细聊？

**Comment by chyroc** [comment]
博主你好，拜读了你的文章，非常棒，令我受益匪浅。

**Comment by LowLowCoder-Mel** [comment]
现在的图片为什么都加载不出来了?

**Comment by LowLowCoder-Mel** [comment]
EOS 几十万貌似不太属实,  有人测试过 6万笔交易 最高1800 平均是 1000

**Comment by draveness** [comment]
> @zhang6321615
> EOS 几十万貌似不太属实,  有人测试过 6万笔交易 最高1800 平均是 1000

确实是这样的，也就几千

**Comment by draveness** [comment]
> @zhang6321615
> 现在的图片为什么都加载不出来了?

忘了申请 SSL 证书，在搞

**Comment by dongguaWDY** [comment]
有几个疑惑，大佬有空帮忙解答下：
1.到底什么是共识算法，业界有个标准定义吗？
2.共识算法和一致性的关系，共识算法就一定能保证强一致吗？共识算法就是强一致算法吗？还是说共识算法是实现强一致目的的重要工具？

**Reply by spade69** [reply]
是两个概念啊，一致性应该是CAP理论里面的一个因素，一致性有强弱和最终一致性，不一定是算法，也可以是工程的方式去保证。 
共识算法是解决拜占庭这个问题，当然它可以强一致也可以弱一致，然后才有Raft，Paxos什么的算法

**Comment by gongqij** [comment]
勘误:`它的出现就是为了解决 Paxos 难以理解并和难以实现的问题`
